src/Xamarin.Android.NamingCustomAttributes/Android.App/ActivityAttribute.cs

```diff
@@ -23,24 +23,32 @@ public ActivityAttribute ()
 #endif
 		public bool                   AllowTaskReparenting    {get; set;}
 		public bool                   AlwaysRetainTaskState   {get; set;}
+		public bool                   AutoRemoveFromRecents   {get; set;}
+		public string                 Banner                  {get; set;}
 		public bool                   ClearTaskOnLaunch       {get; set;}
 #if ANDROID_26
 		public string                 ColorMode               {get; set;}
 #endif
 		public ConfigChanges          ConfigurationChanges    {get; set;}
+		public string                 Description             {get; set;}
 #if ANDROID_24
 		public bool                   DirectBootAware         {get; set;}
 #endif
+		public DocumentLaunchMode     DocumentLaunchMode      {get; set;}
+		public string                 EnableVrMode            {get; set;}
 		public bool                   Enabled                 {get; set;}
 		public bool                   ExcludeFromRecents      {get; set;}
 		public bool                   Exported                {get; set;}
+		public bool                   FinishOnCloseSystemDialogs    {get; set;}
 		public bool                   FinishOnTaskLaunch      {get; set;}
 #if ANDROID_11
 		public bool                   HardwareAccelerated     {get; set;}
 #endif
 		public string                 Icon                    {get; set;}
 		public string                 Label                   {get; set;}
 		public LaunchMode             LaunchMode              {get; set;}
+		public LockTaskMode           LockTaskMode            {get; set;}
```

Comment:
This is not documented at: https://developer.android.com/guide/topics/manifest/activity-element